pallets/stellar-relay/Cargo.toml

```diff
@@ -24,6 +24,7 @@ frame-system = { default-features = false, git = "https://github.com/paritytech/
 frame-benchmarking = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.36", optional = true }
 
 sp-std = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.36" }
+sp-core = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.36" }
```

Comment:
I just noticed; if this is only added just so we could use the `H256` datatype, `spacewalk-primitives` is actually available: 
https://github.com/pendulum-chain/spacewalk/blob/dev/milestone-3/primitives/src/lib.rs#L9

The `H256` is exposed already.

I think we can actually remove `sp-core` altogether.

Reply:
Hmm yes you're right, that's the only reason I added sp-core. But I just checked the other pallets that use H256 and from what I can tell they all import it from sp_core instead of the primitives. Do you think we should change it to use the one from primitives?

Reply:
^ that would be ideal, imo.  One less dependency in the toml files. :eyes:

Reply:
Haha, true there would be one less dependency, but maybe we should try to depend on the spacewalk primitives as little as possible in the crates? Not so sure about it. Anyways, it does not really matter at this point so I'll change it to use the one from primitives.

Reply:
Nevermind, after changing the code to use the one from primitives I realized that the primitives dependency is declared as 'optional' and only included when the 'testing-utils' feature is set...... so I would say we keep it like this 😅